internal/command/root/root.go

```diff
@@ -77,7 +79,13 @@ func New() *cobra.Command {
 		short = "The Fly.io command line interface"
 	)
 
-	root := command.New("fly", short, long, run)
+	exePath, err := os.Executable()
+	if err != nil {
+		panic(err)
```

Comment:
A panic is too much, not sure what are the chances of `os.Executable()` to error, but I would go for `log.Printf("WARN: failed to find executable, error=%q, err)` and default `exe` to `fly`.